tensorflow_addons/losses/BUILD

```diff
@@ -17,6 +18,7 @@ py_library(
     ],
     deps = [
         "//tensorflow_addons/activations",
+        "//tensorflow_addons/layers",
```

Comment:
You'll need to add `//tensorflow_addons/text/` as a dep. Better yet, create a separate target only for `text.crf` in `text/BUILD`, and add that as a dependency.

Reply:
I am not sure what I do is the thing exactly what you want, but I am do updated codes to fix the dependencies issue.